blog/_posts/2019-02-11-julia-entities.md

```diff
@@ -0,0 +1,35 @@
+
+## Julia
+
+The Julia project was founded by [Jeff Bezanson](https://github.com/JeffBezanson), [Alan Edelman](https://en.wikipedia.org/wiki/Alan_Edelman), [Viral Shah](https://en.wikipedia.org/wiki/Viral_B._Shah) and myself ([Stefan Karpinski](https://en.wikipedia.org/wiki/Stefan_Karpinski)). It consists of some code and a community of people who work on that code. The most clear cut line that can be drawn is that there is a set of people who have commit access to the [JuliaLang](https://github.com/JuliaLang) GitHub organization: there are currently 67 committers (36 active and 31 dormant). This set of people doesn’t really define the project, however, since there are many people who are prolific contributors to the Julia ecosystem but who do not have “commit bit.” The communal nature of open source makes it difficult to precisely define where the Julia project ends and the greater community begins, which is exactly how we like it.
+
+## Julia Stewards
+
+One official group of people representing the Julia project does exist: [Julia Stewards](https://julialang.org/community/stewards/). This group exists to handle conflict resolution and reports of inappropriate or problematic behavior in the Julia community and ecosystem. The current stewards are: Milan Bouchet-Valat, Simon Byrne, Tim Holy, Katharine Hyatt, Steven Johnson, Stefan Karpinski, and Viral Shah. This group gets (rare) emails about potentially problematic situations and confidentially discusses and decides on what to do about them. The stewards group is not responsible for making any kinds of technical decisions or steering the project in any other way—it just exists to handle complaints and disputes, which fortunately are exceedingly rare.
+
+## The Julia Lab
+
+A lot of the early work on the development of core Julia was done at MIT in what is now known as [The Julia Lab](https://julia.mit.edu) under the direction of professor [Alan Edelman](https://en.wikipedia.org/wiki/Alan_Edelman). All of the co-creators of Julia have been part of the lab at some point. In those early days, we didn’t call it the Julia Lab yet, it was just an office where some people worked on high performance computing, numerical linear algebra, [random matrix theory](https://en.wikipedia.org/wiki/Random_matrix), and this new programming language called Julia. Although not formally part of the Julia Lab, Dr Jeremy Kepner of Lincoln Laboratory was an early believer in and funder of the lab’s work on Julia and continues to support this work. Since those early days, things have gotten a lot more organized under the leadership (in chronological order) of [Jiahao Chen](https://jiahao.github.io), [Andreas Noack](https://github.com/andreasnoack) and [Valentin Churavy](https://github.com/vchuravy), and the Julia Lab remains a steady source of major innovations and contributions to Julia.
```

Comment:
```suggestion
A lot of the early work on the development of core Julia was done at MIT in what is now known as [The Julia Lab](https://julia.mit.edu) under the direction of professor Alan Edelman. All of the co-creators of Julia have been part of the lab at some point. In those early days, we didn’t call it the Julia Lab yet, it was just an office where some people worked on high performance computing, numerical linear algebra, [random matrix theory](https://en.wikipedia.org/wiki/Random_matrix), and this new programming language called Julia. Although not formally part of the Julia Lab, [Jeremy Kepner](https://www.ll.mit.edu/biographies/jeremy-kepner) of Lincoln Laboratory was an early believer in and funder of the lab’s work on Julia and continues to support this work. Since those early days, things have gotten a lot more organized under the leadership (in chronological order) of [Jiahao Chen](https://jiahao.github.io), [Andreas Noack](https://github.com/andreasnoack) and [Valentin Churavy](https://github.com/vchuravy), and the Julia Lab remains a steady source of major innovations and contributions to Julia.
```